hasjob/static/js/app.js

```diff
@@ -86,14 +86,18 @@ window.Hasjob.Currency.indian_rupee_encoder = function(value) {
 
 window.Hasjob.Currency.prefix = function(currency){
   var currencyMap = {
-    'na': '¤',
+    'default': '¤',
     'inr': '₹',
     'usd': '$',
     'sgd': '$',
```

Comment:
Can we fix the code to allow multi-character prefixes? So SGD becomes `S$` (and similarly, AUD `A$`). That'll match convention. As I recall there's code elsewhere to ignore the first character when converting a formatted number back to an integer.